modules/widgets/follow-button.php

```diff
@@ -26,6 +26,21 @@ public function widget( $args, $instance ) {
 		$attributes = array();
 		$instance = wp_parse_args( (array) $instance, array( 'show_name' => 1, 'show_count' => 0 ) );
 
+		$wpcom_locale = get_locale();
+
+		if ( ! class_exists( 'GP_Locales' ) ) {
+			if ( defined( 'JETPACK__GLOTPRESS_LOCALES_PATH' ) && file_exists( JETPACK__GLOTPRESS_LOCALES_PATH ) ) {
```

Comment:
cc-ing @georgestephanis on this, since he had some concerns and suggestions about our use of `jetpack_require_lib ` in #5097.